lib/node_modules/@stdlib/fs/resolve-parent-paths-by/docs/repl.txt

```diff
@@ -0,0 +1,142 @@
+
+{{alias}}( paths[, options], predicate, clbk )
+    Asynchronously resolves paths from a set of paths according to a
+    predicate function by walking parent directories.
+
+    When invoked, the predicate function is provided two arguments:
+
+    - `path`: resolved paths
+    - `next`: a callback to be invoked after processing a resolved path
+
+    The `next` callback takes two arguments:
+
+    - `error`: error argument
+    - `result`: test result
+
+    If a provided predicate function calls the `next` callback with a truthy
+    `error` argument, the function suspends execution and immediately calls
+    the `done` callback for subsequent `error` handling.
+
+    The function immediately returns upon encountering a non-falsy `result`
+    value and calls the `done` callback with `null` as the first argument and
+    the array of resolved paths as the second argument.
+
+    If unable to resolve paths, the function returns empty array as the path
```

Comment:
Not the case in `each` mode. Better to be explicit here. Also update the `README` accordingly ([ref](https://github.com/stdlib-js/stdlib/pull/2897#discussion_r1761644554)).